chef_master/source/supermarket_backup_restore.rst

```diff
@@ -3,21 +3,86 @@ Supermarket Backup and Restore
 
-**Cookbooks**
 
-The default location of cookbooks is here: `/var/opt/supermarket/data`
+Backup
+=====================================================
+
+Cookbook Backup
+-----------------------------------------------------
+
+If you are not using AWS S3 storage for cookbooks, you will also need to make a backup of the local cookbook storage location. 
+
+The default location is: ``/var/opt/supermarket/data/cookbook_versions``.
+
+For example, a cookbook backup command:
+
+.. code-block:: bash
+
+     cd /var/opt/supermarket/data/ 
+     tar cvzf supermarket_cookbook_versions.tar.gz cookbook_versions
+
+Database Backup
+-----------------------------------------------------
+
+A database export can be made in several formats. 
+
+For example, a database export in a .dump format can be made with the following syntax:
+
+.. code-block:: bash
+
+     $ /opt/supermarket/embedded/bin/pg_dump -h localhost -U supermarket -d supermarket_production -p 15432 -F c -b -v -f smkt-date-blah-backup.dump
+
+where, in a typical installation:
+  * ``/opt/supermarket/embedded/bin/pg_dump`` is
+  * ``localhost`` may alternatively be 127.0.0.1 
+  * ``15432`` is PostgreSQL port number, which may need to be modified.
+  * ``-F c`` makes a custom .dump format
+
+For documentation about the pg_dump utility, see: https://www.postgresql.org/docs/9.3/app-pgdump.html
+
+Be sure to update the various local values in the syntax as necessary to match your infrastructure. To find local variables, look at ``/var/opt/supermarket/etc/database.yml``. For example, an output of ``database.yml`` may look like:
+
+.. code-block:: yaml
+
+     production:
+         adapter: postgresql
+         database: supermarket
+         username: supermarket
+         password:
+         host: 127.0.0.1
+         port: 15432
+         pool: 25
+
+
+Restore
+=====================================================
+
+Cookbook Restore
+-----------------------------------------------------
+
+When restoring cookbooks, **make sure the cookbook directory is writable by the supermarket user.**
+
+For example, to restore your cookbook files, run:
+
+.. code-block:: bash
+
+     cd /var/opt/supermarket/data/
+      tar xvfz /path/to/supermarket_cookbook_versions.tar.gz
+
+
+Database Restore
+-----------------------------------------------------
+
+.. note::
+
+    The restore does not support transferring backups across different versions of Supermarket. Backups taken must be restored to the same version of Supermarket that was in use when they were created.
 
-**Database**
+For example, to restore a backup in a .dump format, run: 
 
-A database export can be make with the following syntax:
+.. code-block:: bash
 
-```
-/opt/supermarket/embedded/bin/pg_dump <supermarket_database_name> -c -f <targetfilename>`
-```
+     $ pg_restore -h localhost -p 15432 --clean --no-acl --no-owner -d supermarket_production -v smkt-date-blah-backup.dump
```

Comment:
Long-form switches to ~reduce arcanitiousness~ ... ~reduce occultiness~ ... make it clearerer.

```suggestion
     $ pg_restore --host localhost --port 15432 --clean --no-acl --no-owner --dbname supermarket_production --verbose supermarket_database_backup.dump
```